src/main/java/magicbees/main/utils/compat/thaumcraft/NodeHelper.java

```diff
@@ -109,6 +118,9 @@ public static boolean growNodeInRange(List<Chunk> chunks, World world, int xCoor
         for (Chunk chunk : chunks) {
             for (INode node : findNodesInChunkWithinRange(chunk, xCoord, yCoord, zCoord, range)) {
                 if (node != null) {
+                    if (ThaumicHorizonsHelper.isActive() && node instanceof TileSyntheticNode) {
+                        continue;
+                    }
```

Comment:
Check is not necessary here. `findNodesInChunkWithRange` preforms it already.

Reply:
lmfao fair, I was going for overkill